pkg/cmd/clustertask/start.go

```diff
@@ -149,7 +149,7 @@ func startClusterTask(opt startOptions, args []string) error {
 	}
 
 	var ctname string
-	timeoutSeconds := time.Duration(opt.TimeOut) * time.Second
+	timeoutSeconds, _ := time.ParseDuration(opt.TimeOut)
```

Comment:
You'll want to check if an error happens during duration here. I would also recommend renaming the variable since the timeout is no longer specified in seconds:

```go
timeout, err := time.ParseDuration(opt.TimeOut)
if err != nil {
    return err
}
```

This would also require you to change the variable name where it is input to the `TaskRun`:

https://github.com/tektoncd/cli/blob/216f375412796f6c1086a31cb74e85ba212ff430/pkg/cmd/clustertask/start.go#L160